providers/ytt/03_customizations/pod_security_admission.yaml

```diff
@@ -0,0 +1,56 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+#@ load("@ytt:base64", "base64")
+#@ load("@ytt:yaml", "yaml")
+#@ load("@ytt:version", "version")
+#@ load("/lib/helpers.star", "enable_pod_security_standard")
+
+#@ def kube_apiserver_admission_pss():
+apiVersion: apiserver.config.k8s.io/v1
+kind: AdmissionConfiguration
+plugins:
+- name: PodSecurity
+  configuration:
+    apiVersion: pod-security.admission.config.k8s.io/v1beta1
+    kind: PodSecurityConfiguration
+    defaults:
+      enforce: #@ data.values.POD_SECURITY_STANDARD_ENFORCE
+      enforce-version: "latest"
+      audit: #@ data.values.POD_SECURITY_STANDARD_AUDIT
+      audit-version: "latest"
+      warn: #@ data.values.POD_SECURITY_STANDARD_WARN
+      warn-version: "latest"
```

Comment:
I think we'll want to pin to a specific version, instead of just using `latest`.

Reply:
TODO by me. I need to take another look at it.